server/plugin/command.go

```diff
@@ -289,6 +291,10 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		} else if len(optionList) == 1 {
 			features = optionList[0]
 			fs := strings.Split(features, ",")
+			if SliceContainsString(fs, "issues") && SliceContainsString(fs, "issue_creations") {
```

Comment:
Nit pick: Let's use the `const`s
```suggestion
			if SliceContainsString(fs, featureIssues) && SliceContainsString(fs, featureIssueCreation) {
```